<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="appRouter">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="streamingPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="previewContainer" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="controlContainers" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <widget class="QPushButton" name="startButton">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>30</y>
            <width>141</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Start Streaming</string>
          </property>
         </widget>
         <widget class="QLabel" name="instructionLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>401</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Connect from client with these address:</string>
          </property>
         </widget>
         <widget class="QLabel" name="ip_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>391</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>IP: N/A</string>
          </property>
         </widget>
         <widget class="QLabel" name="port_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>381</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>PORT: N/A</string>
          </property>
         </widget>
         <widget class="QLabel" name="connected_status_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>341</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Current Status: Not Connected</string>
          </property>
         </widget>
         <widget class="QLabel" name="frameRateLabel">
          <property name="geometry">
           <rect>
            <x>840</x>
            <y>20</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Frame Rate:</string>
          </property>
         </widget>
         <widget class="QComboBox" name="frameRateComboBox">
          <property name="geometry">
           <rect>
            <x>910</x>
            <y>20</y>
            <width>69</width>
            <height>22</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>60</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="adapterLabel">
          <property name="geometry">
           <rect>
            <x>840</x>
            <y>50</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Adapter: </string>
          </property>
         </widget>
         <widget class="QLabel" name="outputLabel">
          <property name="geometry">
           <rect>
            <x>840</x>
            <y>80</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Output:</string>
          </property>
         </widget>
         <widget class="QComboBox" name="adapterComboBox">
          <property name="geometry">
           <rect>
            <x>910</x>
            <y>50</y>
            <width>69</width>
            <height>22</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>NULL</string>
          </property>
          <property name="duplicatesEnabled">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QComboBox" name="outputComboBox">
          <property name="geometry">
           <rect>
            <x>910</x>
            <y>80</y>
            <width>69</width>
            <height>22</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>NULL</string>
          </property>
          <property name="duplicatesEnabled">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="settingsButton">
          <property name="geometry">
           <rect>
            <x>560</x>
            <y>30</y>
            <width>51</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
         <widget class="QLabel" name="connected_status_label_2">
          <property name="geometry">
           <rect>
            <x>840</x>
            <y>120</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tip: Add virtual Monitor at settings</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settingsPage">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
         <widget class="QPushButton" name="backToStreaming">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
         <widget class="QPushButton" name="aboutSettingsButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>About</string>
          </property>
         </widget>
         <widget class="QPushButton" name="virtualMonitorSettingsButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Virtual Monitors</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="settingsRouter">
         <widget class="QWidget" name="aboutSettings">
          <widget class="QLabel" name="aboutLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>About</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>681</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>Screen Duplicator is a software that streams a virtual second scrreen to another device in partner with Screen Duplicator Client.
This program uses c++, qt for UI, dxgi for screen duplication, and ffmpeg for compression. The project is Open-Source so
 feel free to contribute to the github repo linked below.</string>
           </property>
          </widget>
          <widget class="QLabel" name="aboutLabel_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>250</y>
             <width>71</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Socials</string>
           </property>
          </widget>
          <widget class="QPushButton" name="githubRepoButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>160</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Github Repo</string>
           </property>
          </widget>
          <widget class="QPushButton" name="clientGithubRepoButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Client Github Repo</string>
           </property>
          </widget>
          <widget class="QPushButton" name="linkedinButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>290</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Linkedin</string>
           </property>
          </widget>
          <widget class="QPushButton" name="githubButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>330</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Github</string>
           </property>
          </widget>
          <widget class="QPushButton" name="facebookButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>370</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Facebook Page</string>
           </property>
          </widget>
          <widget class="QPushButton" name="websiteButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>410</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Website</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="virtualMonitorSettings">
          <widget class="QLabel" name="VirtualMonitorLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Virtual Monitors</string>
           </property>
          </widget>
          <widget class="QLabel" name="virtualMonitorNote">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>731</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>NOTE: This software uses unmodified Amyuni USB Mobile Monitor Virtual Display drivers.   </string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
          </widget>
          <widget class="QPushButton" name="amyuniInstallButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>180</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Install </string>
           </property>
          </widget>
          <widget class="QPushButton" name="amyuniUninstallButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>260</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Uninstall</string>
           </property>
          </widget>
          <widget class="QPushButton" name="amyuniAddMonitorButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>340</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Add Virtual Monitor</string>
           </property>
          </widget>
          <widget class="QPushButton" name="AmyuniRemoveMonitorButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>420</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Remove Virtual Monitor</string>
           </property>
          </widget>
          <widget class="QLabel" name="virtualMonitorNote_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>391</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Install Amyuni USB Mobile Monitor Virtual Display</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
          </widget>
          <widget class="QLabel" name="virtualMonitorNote_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>230</y>
             <width>391</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Uninstall Amyuni USB Mobile Monitor Virtual Display</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
          </widget>
          <widget class="QLabel" name="virtualMonitorNote_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>310</y>
             <width>411</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Create a Virtual Monitor using Amyuni USB Mobile Monitor Virtual Display</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
          </widget>
          <widget class="QLabel" name="virtualMonitorNote_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>390</y>
             <width>431</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Remove a Virtual Monitor using Amyuni USB Mobile Monitor Virtual Display</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
